src/en/news/blog/2022/install-ceph-in-a-raspberrypi-4-cluster/index.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Install Ceph in a Raspberry Pi 4 Cluster
+date: 2022-02-16
+image: images/raspberrypi-cluster.jpg
+author: Mike Perez (thingee)
+---
```

Comment:
BTW, I'd add some tags here: raspberry_pi, arm, tutorial, pacific, cephadm, dashboard, ... And if you plan to deliver more posts about this topic, maybe some common tag for these..

Reply:
Yes there will be a series of posts. I will add more tags. :)